Makefile.in

```diff
@@ -203,16 +203,33 @@ uninstall:
 	rm -f $(DESTDIR)$(datarootdir)/bash-completion/completions/firecfg
 	@echo "If you want to install a different version of firejail, you might also need to run 'rm -fr $(DESTDIR)$(sysconfdir)/firejail', see #2038."
 
-DISTFILES = "src etc m4 platform contrib configure configure.ac Makefile.in install.sh mkman.sh mketc.sh mkdeb.sh.in COPYING README RELNOTES"
-DISTFILES_TEST = "test/Makefile.in test/apps test/apps-x11 test/apps-x11-xorg test/root test/private-lib test/fnetfilter test/fcopy test/environment test/profiles test/utils test/compile test/filters test/network test/fs test/sysutils test/chroot"
+DISTFILES = \
+COPYING \
+Makefile \
```

Comment:
the `dist` target is creating the tarball that is released after a new firejail version is released.  
normally release tarballs don't contain generated files like Makefiles (or mkdeb.sh), as they likely can't be used by someone downloading the tarball (because they contain paths of the system where it was generated, or other configuration options).
(a common exception to that is the pre-generated `configure` script, though in theory it also would not need to be shipped.)  

i would prefer to keep a "clean" release tarball.

Reply:
I'll look into this.

Reply:
@reinerh on [May 21](https://github.com/netblue30/firejail/pull/5154#discussion_r878673602):

> the `dist` target is creating the tarball that is released after a new
> firejail version is released. normally release tarballs don't contain
> generated files like Makefiles (or mkdeb.sh), as they likely can't be used by
> someone downloading the tarball (because they contain paths of the system
> where it was generated, or other configuration options). (a common exception
> to that is the pre-generated `configure` script, though in theory it also
> would not need to be shipped.)
> 
> i would prefer to keep a "clean" release tarball.

Yes, I don't consider it good either; I just didn't manage to think of
something better when opening this PR.  As of the current version, this is not
done anymore.

Related to this, I now see that as of #5142, since the entirety of src/ and
test/ are part of `DISTFILES` / `DISTFILES_TEST` (rather than only the exact
files and dirs that are needed), the generated `Makefile` files in these
directories are also being copied on `make dist` (besides just the
`Makefile.in` files).  But that should be fixed with the PR that will come
after #5140.

Reply:
> Related to this, I now see that as of #5142, since the entirety of src/ and
> test/ are part of `DISTFILES` / `DISTFILES_TEST` (rather than only the exact
> files and dirs that are needed), the generated `Makefile` files in these
> directories are also being copied on `make dist` (besides just the
> `Makefile.in` files). But that should be fixed with the PR that will come
> after #5140.

Actually, only src/ is affected, as test/Makefile.in is referenced directly in
`DISTFILES_TEST` and there is no other makefile in test/.

Fixed by #5182.